apitools/base/py/exceptions.py

```diff
@@ -143,14 +182,17 @@ class RetryAfterError(HttpError):
 
     """The response contained a retry-after header."""
 
-    def __init__(self, response, content, url, retry_after):
-        super(RetryAfterError, self).__init__(response, content, url)
+    def __init__(self, response, content, url, retry_after,
+                 method_config=None, request=None):
+        super(RetryAfterError, self).__init__(
+            response, content, url, method_config, request)
         self.retry_after = int(retry_after)
 
     @classmethod
-    def FromResponse(cls, http_response):
+    def FromResponse(cls, http_response, method_config=None, request=None):
         return cls(http_response.info, http_response.content,
-                   http_response.request_url, http_response.retry_after)
+                   http_response.request_url, http_response.retry_after,
+                   method_config, request)
```

Comment:
I'd use explicit names for the last two args (though my previous comment will make this one irrelevant).

Reply:
No longer relevant.